monitoring/variables.tf

```diff
@@ -20,4 +20,5 @@ locals {
   paas_api_url             = "https://api.london.cloud.service.gov.uk"
   monitoring_space_name    = "bat-${var.monitoring_env}"
   monitoring_instance_name = var.monitoring_env == "prod" ? "bat" : "bat-${var.monitoring_env}"
+  influxdb_service_plan    = var.monitoring_env == "prod" ? "small-1_x" : "tiny-1_x"
```

Comment:
Can we move this to a new tfvars file instead?
monitoring_instance_name could go as well

Reply:
@saliceti done

Reply:
Thanks 👍